Added a term entry for is-nan in PHP (#2087) (#2126)
Codecademy/docs: content/php/concepts/math-functions/terms/is-nan/is-nan.md

---

> Title: 'is-nan' 

Reviewer: ```suggestion
Title: 'is-nan()'
```

> Title: 'is-nan' 

Reviewer: ```suggestion
Title: 'is-nan()'
```

> Description: 'Finds whether a value is not a number.'

Reviewer: ```suggestion
Description: 'Returns a boolean indicating if a value is not a number.'
```

Subjects:
  - 'Computer Science'
  - 'Web Design'
  - 'Web Development'
Tags:
  - 'Functions'
  - 'Arithmetic'

>   - 'An nth tag'

Reviewer: ```suggestion
```

CatalogContent: 
  - 'learn-php'
  - 'paths/computer-science'
---

> The **`is_nan`** function checks whether the argument is 'not a number'.

Reviewer: ```suggestion
The **`is_nan()`** function checks whether the argument is not a number.
```

## Syntax

```pseudo
$result = is_nan(n);
```

> Returns **true** if `n` is 'not a number', else **false**.

Reviewer: ```suggestion
Returns `true` if `n` is 'not a number', else `false`.
```

Reviewer: It looks like when it's false, it returns nothing and returns 1 when true

## Codebyte Example

> Using `is_nan()` to check whether `10` is 'not a number':

Reviewer: ```suggestion
Using `is_nan()` to check whether `10` is not a number:
```

```codebyte/php
<?php
  echo(is_nan(10));

> ?>

Reviewer: ```suggestion
?>
```
```

Reviewer: Nothin should be echoed because it's false

---

> Title: 'is-nan()' 

Reviewer: ```suggestion
Title: 'is_nan()'
```

Description: 'Checks whether a value is not a number.'
Subjects:
  - 'Computer Science'
  - 'Web Design'
  - 'Web Development'
Tags:
  - 'Functions'
  - 'Arithmetic'

>   

Reviewer: ```suggestion
```

CatalogContent: 
  - 'learn-php'
  - 'paths/computer-science'
---

The **`is_nan()`** function checks whether the argument is not a number.

## Syntax

```pseudo
$result = is_nan(n);
```

Returns `1` if `n` is not a number, else returns nothing.

## Codebyte Example

Using `is_nan()` to check whether `10` is 'not a number':

```codebyte/php
<?php

> echo "10:", is_nan(10) . "<br>";
> echo "acos 1.01: ", is_nan(acos(1.01));

Reviewer: ```suggestion
  echo "10:", is_nan(10), "\n";
  echo "sqrt of -1: ", is_nan(sqrt(-1));
```

Reviewer: I like this example, since sort of a negative number is undefined
